src/Airy/airy.jl

```diff
@@ -0,0 +1,299 @@
+        p1 = evalpoly(1 / x3, (-1.5707963267948966, -0.22907446432425577, 0.15510250188621483))
+        xsqr = sqrt(sqrt(x))
+        ai = p / (PIPOW3O2(T) * xsqr)
+        aip = p1 * xsqr / PIPOW3O2(T) 
+        return ai, aip
+    else
+        inv_sqpi = 5.64189583547756286948e-1 # 1/sqrt(pi)
+        xsqr = sqrt(x)
+        zinv = 3 / (2 * x * xsqr)
+        xsqr = sqrt(xsqr)
+        p = evalpoly(zinv, (9.99999999999999995305e-1, 1.40264691163389668864e1, 7.05360906840444183113e1, 1.59756391350164413639e2, 1.68089224934630576269e2, 7.62796053615234516538e1, 1.20075952739645805542e1, 3.46538101525629032477e-1)) 
+        q = evalpoly(zinv, (1.00000000000000000470e0, 1.40959135607834029598e1, 7.14778400825575695274e1, 1.64234692871529701831e2, 1.77318088145400459522e2, 8.45138970141474626562e1, 1.47562562584847203173e1, 5.67594532638770212846e-1))
+        p1 = evalpoly(zinv, (1.00000000000000000550e0, 1.39470856980481566958e1, 6.99778599330103016170e1, 1.59317847137141783523e2, 1.71184781360976385540e2, 8.20584123476060982430e1, 1.47454670787755323881e1, 6.13759184814035759225e-1))
+        q1 = evalpoly(zinv, (9.99999999999999994502e-1, 1.38498634758259442477e1, 6.86752304592780337944e1, 1.53206427475809220834e2, 1.58778084372838313640e2, 7.11727352147859965283e1, 1.11810297306158156705e1, 3.34203677749736953049e-1))
+        ai = inv_sqpi * p / (q * 2 * xsqr)
+        aip = -0.5 * inv_sqpi * xsqr * p1 / q1
+        return ai, aip
+    end
+end
+    
+function airyai_large_neg_arg(x::T) where T <: Float64
+    x3 = x * x * x
+    p = evalpoly(1 / x3, (1.5707963267948966, 0.13124057851910487, 0.4584353787485384, 5.217255928936184, 123.97197893818594, 5038.313653002081, 312467.7049060495, 2.746439545069411e7, 3.2482560591146026e9, 4.97462635569055e11, 9.57732308323407e13))
+    q = evalpoly(1 / x3, (0.1636246173744684, 0.20141783231057064, 1.3848568733028765, 23.555289417250567, 745.2667344964557, 37835.063701047824, 2.8147130917899106e6, 2.8856687720069575e8, 3.8998976239149216e10, 6.718472897263214e12, 1.4370735281142392e15))
+    p1 = evalpoly(1 / x3, (-1.5707963267948966, 0.15510250188621483, 0.4982993247266722, 5.515384839161109, 129.24738229725767, 5209.103946324185, 321269.61208650155, 2.812618811215662e7, 3.3166403972012258e9, 5.0676100258903735e11, 9.738286496397669e13))
+    q1 = evalpoly(1 / x3, (0.22907446432425577, 0.22511404787652015, 1.4803642438754887, 24.70432792540913, 773.390007496322, 38999.21950723391, 2.8878225227454924e6, 2.950515261265541e8, 3.97712331943799e10, 6.837383921993536e12, 1.460066704564067e15))
+
+    sq_x3 = im*sqrt(abs(x3))
+    c1 = exp(2 * sq_x3 / 3) # loses some precision here
+    a = muladd(1 / sq_x3, -q, p) / c1
+    b = muladd(1 / sq_x3, q, p) * c1
+    c = muladd(1 / sq_x3, -q1, p1) / c1
+    d = muladd(1 / sq_x3, q1, p1) * c1
+
+    xsqrt = sqrt(sqrt(complex(x)))
+    ai = real((b + im*a) / (PIPOW3O2(T) * xsqrt))
+    aip = imag((c + im*d) * xsqrt  / PIPOW3O2(T))
```

Comment:
these seem overly complicated.

Reply:
Yes you are right. Because just on real-line can simplify to
```julia
ai = 2*real(a*im / (PIPOW3O2(T) * xsqrt))
aip = -2*real(c*im * xsqrt  / PIPOW3O2(T))
```
And can probably make some more simplifications since we only need real part